process_onestop_sr_report/preprocessing.py

```diff
@@ -687,7 +699,10 @@ def add_word_metrics(df: pd.DataFrame, args: ArgsParser) -> pd.DataFrame:
         spacy_model_name=args.NLP_MODEL,
         add_question_in_prompt=args.add_question_in_prompt,
         parsing_mode=args.parsing_mode,
+        model_target_device=args.device,
+        hf_access_token=args.hf_access_token,
```

Comment:
this won't work with older versions of text-metrics. maybe specify in yaml minimum version or exact (new) version